scripts/changelog_check.rb

```diff
@@ -112,14 +127,11 @@ def generate_changelog(git_log)
     next if item.commit_messages.any? { |message| message.include?(SKIP_CHANGELOG_MESSAGE) }
     change = build_changelog_from_commit(item)
     next unless change
+    category = closest_change_category(change)
+    next unless category
```

Comment:
> should we warn or error if we can't find a close category?

Assuming the developer tried to add a changelog but this failed to detect a closest option, it would still fail the build with the default error message. Do you think we ought to have something more specific than that for this scenario?

Reply:
oh I thought this was the script to generate the output, not to test in CI --- if it fails, that's good enough for me!

Reply:
Technically it's used in both the test and the output generation, but yeah, will be caught in CI.